valheim_plus.cfg

```diff
@@ -66,6 +66,18 @@ noWeatherDamage=false
 ; The maximum range that you can place build objects at
 maximumPlacementDistance=5
 
+; When destroying a building piece, setting this to true will ensure it always drops full resources.
+alwaysDropResources=false
+
+; When destroying a building piece, setting this to true will ensure it always drops pieces that the devs have marked as "do not drop".
```

Comment:
I wonder if we should add some examples of some pieces that are considered "do not drop"?

Reply:
I actually don't know exactly what things are marked as "do not drop". For some context, both of these options were created because I often found, when playing online with my partner, that sometimes destroying buildings would not refund all of the materials. I know a combination of both has eliminated that problem for us. If you/anyone has a good method of scanning the game data, looking for the field `m_recover` in any piece requirements, we could generate a list...

Reply:
@Mixone-FinallyHere might know of a way to do this but for now it's fine as is